release/release.sh

```diff
@@ -164,25 +198,18 @@ banner "Creating and pushing branch ${NEW_BRANCH}"
 run git checkout -b "${NEW_BRANCH}" "${NEW_TAG}"
 run git push --set-upstream origin "${NEW_BRANCH}"
 
-# Maybe todo: extract the release notes from the CHANGELOG.md file and stick
-#             them in the release body.
+banner "Using release notes for ${NEW_TAG}"
+readonly RELEASE_NOTES="$(get_release_notes "${NEW_TAG}")"
+echo "${RELEASE_NOTES}"
+
 banner "Creating release"
 run hub release create \
   --prerelease \
-  --message="${NEW_TAG} Release" \
-  --message="*Paste release notes here*" \
+  --file=<(printf "%s\n\n%s" "${NEW_TAG} Release" "${RELEASE_NOTES}") \
   "${NEW_TAG}"
 
 banner "Success!"
-readonly release_fmt="
-   date: %cI
-    url: %U
-tarball: %uT
-zipball: %uZ
-  state: %S
-  title: %t
-   body: %b
-"
+readonly release_fmt="%n date: %cI %n  url: %U %nstate: %S %ntitle: %t %n"
```

Comment:
Do you like the line-ending whitespace?

Reply:
Yes, I did like them because I think they made the format string easier to read. But removed them in https://github.com/googleapis/google-cloud-cpp/pull/3760